crates/witgen_macro_helper/src/generator.rs

```diff
@@ -7,15 +7,15 @@ use syn::{Attribute, ItemEnum, ItemFn, ItemStruct, ItemType, Lit, ReturnType, Ty
 
 use crate::{is_known_keyword, ToWitType};
 
-pub(crate) fn get_target_dir() -> PathBuf {
+pub fn get_target_dir() -> PathBuf {
     let metadata = MetadataCommand::new()
         .exec()
         .expect("cannot fetch cargo metadata");
 
     metadata.target_directory.join("witgen").into()
 }
 
-pub(crate) fn gen_wit_struct(strukt: &ItemStruct) -> Result<String> {
+pub fn gen_wit_struct(strukt: &ItemStruct) -> Result<String> {
```

Comment:
Could you add some documentation over these gen_* functions ? using `/// the documentation`